data/json/recipes/food/dry.json

```
@@ -11,7 +11,7 @@
     "autolearn": true,
     "batch_time_factors": [ 67, 5 ],
     "tools": [ [ [ "dehydrator", 25 ], [ "char_smoker", 25 ] ] ],
-    "components": [ [ [ "fish", 1 ] ] ]
+    "components": [ [ [ "fish", 1 ], [ "fish_scrap", 1 ] ] ]
```

```suggestion
    "components": [ [ [ "fish", 1 ], [ "fish_scrap", 10 ] ] ]
```
Looks like a typo, I think you meant to use 10 here

---

Yep! Sorry!